cypress/TestCases/FireboltCertification/Manage/KeyboardManage_RpcOnly.feature

```diff
@@ -31,7 +31,7 @@ Feature: Keyboard_Manage_RpcOnly
         Then 'Firebolt' platform responds with '<MethodResponse>'
 
         Examples:
-            | Scenario              | ManageEventName                   | EventName                  | MethodName        | MethodParam                                              | FocusMethod                       | MethodResponse        |
-            | Validating rpc method | keyboard onRequestEmail manage    | keyboard onRequestEmail    | keyboard email    | prompt the user for their email address with signin type | send error from email provider    | entered email address |
-            | Validating rpc method | keyboard onRequestStandard manage | keyboard onRequestStandard | keyboard standard | prompt the user for their username                       | send error from standard provider | entered username      |
-            | Validating rpc method | keyboard onRequestPassword manage | keyboard onRequestPassword | keyboard password | prompt the user for their password                       | send error from password provider | entered password      |
\ No newline at end of file
+            | Scenario              | ManageEventName                   | EventName                  | MethodName        | MethodParam                                              | FocusMethod                       | MethodResponse                        |
+            | Validating rpc method | keyboard onRequestEmail manage    | keyboard onRequestEmail    | keyboard email    | prompt the user for their email address with signin type | send error from email provider    | entered email address provider error  |
+            | Validating rpc method | keyboard onRequestStandard manage | keyboard onRequestStandard | keyboard standard | prompt the user for their username                       | send error from standard provider | entered username provider error       |
+            | Validating rpc method | keyboard onRequestPassword manage | keyboard onRequestPassword | keyboard password | prompt the user for their password                       | send error from password provider | entered password provider error       |
```

Comment:
Just use **custom error** in the feature file

Reply:
updated to custom error